pages/platform/databases/information_05_capabilities/guide.en-gb.md

```diff
@@ -0,0 +1,105 @@
+
+### [Redis](/publiccloud/databases/redis/capabilities/)
+Redis is an open-source fully managed in-memory NoSQL database. It's help you store and access data quickly and efficiently.
+
+### [M3db](/publiccloud/databases/m3db/capabilities/)
+M3db is a fully managed distributed time series database. It's scalable and high-availability, ideal for your monitoring environment and time series applications.
+
+### [Opensearch](/publiccloud/databases/opensearch/capabilities/)
+Opensearch is an open-source fully managed search, analytics and observability engine. Ideal for logs management and offer search and analytical aggregations. 
+OpenSearch is an open source fork derived from Elasticsearch.
+
+### [Cassandra](/publiccloud/databases/cassandra/capabilities/)
+Apache Cassandra is an open-source fully managed NoSQL database. It's highly fault tolerant with asynchronous replication.
+
+## Feature Availability
+
+| Feature                                                                                                      | MySQL | PostgreSQL | MongoDB          | Redis | M3db | Opensearch | Cassandra |
+|--------------------------------------------------------------------------------------------------------------|-------|------------|------------------|-------|------|------------|-----------|
+| End-to-end security ([more information](/publiccloud/databases/concepts-security-overview/))                 | ✓     | ✓          | ✓                | ✓     | ✓    | ✓          | ✓         |
+| Automatic updates and one click upgrade                                                                      | ✓     | ✓          | ✓                | ✓     | ✓    | ✓          | ✓         |
+| Read-only nodes                                                                                              | ✓     | ✓          | ✓                |       |      |            |           |
+| Point in Time Recovery                                                                                       | ✓     | ✓          | ✓ <sup>(1)</sup> |       |      |            |           |
+| Database forking                                                                                             | ✓     | ✓          | ✓                | ✓     | ✓    | ✓          | ✓         |
+| Terraform Provider support ([more information](https://registry.terraform.io/providers/ovh/ovh/latest/docs)) | ✓     | ✓          | ✓                | ✓     | ✓    | ✓          | ✓         |
+| Private network connectivity (vRack)                                                                         | ✓     | ✓          | ✓                | ✓     | ✓    | ✓          | ✓         |
+
+(1) depend on the plan
+
+# Specific feature Plans
+
+Three plans are available:
+
+### Essential
+The Essential plan typically offers the basic features and functionalities required for managing a database. 
+It is designed for tests, development environments and non-critical projects.
```

Comment:
Suggestion: "It is designed for test and development environments or non-critical projects."